
time stamp
bool TrackDetect
int32 LeftTrack
int32 RightTrack
bool LeftMarker
bool RightMarker
int32 SelectedTrack
int32[16] InternalSensors
int32 GyroX
int32 GyroY
int32 GyroZ
float32 ErrorLength

